DEVELOPER.md

```
@@ -0,0 +1,169 @@
+# Event Gate—for Developers
+
+- [Get Started](#get-started)
+- [Set Up Python Environment](#set-up-python-environment)
+- [Run Static Code Analysis](#running-static-code-analysis)
+- [Run Black Tool Locally](#run-black-tool-locally)
+- [Run mypy Tool Locally](#run-mypy-tool-locally)
+- [Run Unit Test](#running-unit-test)
+- [Run Action Locally](#run-action-locally)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)